packages/holocron-module-register-webpack-plugin/package.json

```diff
@@ -28,8 +28,8 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "webpack": "^4.29.5",
-    "webpack-sources": "^1.3.0"
+    "webpack": "^5.88.2",
+    "webpack-sources": "^3.2.3"
   },
```

Comment:
we should consider moving to dev dependencies and peer dependencies. On the face of it the changset in the plugin appears like it should still be compatible webpack 4.

Reply:
Its a bit of a risk with `--use-legacy-peerdeps`

when people come to install the latest bundler, and they get peer dep issues, then just override them, this will fall over